### PR TITLE
Capitalize the lemmas for a whole bunch of demonyms, holidays, and re…

### PR DESCRIPTION
**Pull request by AngledLuffa**
Capitalize the lemmas for a whole bunch of demonyms, holidays, and religious adjectives

### CODE REVIEW

**Comment by AngledLuffa** [comment]
Haven't done "Islamist" yet.  I thought adding the scripts you sent which are useful tools might make it easier for future editors.  I can remove them if you don't like that idea (or put it in a separate change if you want a cleaner git history)

**Comment by nschneid** [comment]
Personally I don't mind including the utility scripts but I worry they may not be wanted in the master branch, and therefore should be kept out of dev as well? @dan-zeman?

**Comment by amir-zeldes** [comment]
> Personally I don't mind including the utility scripts but I worry they may not be wanted in the master branch, and therefore should be kept out of dev as well? @dan-zeman?

Split/merge source files is a useful generic function to have, so maybe it should be included in UD_tools?

**Comment by AngledLuffa** [comment]
This version isn't too generic though, unless all the repos have the same
structure

**Comment by dan-zeman** [comment]
> Personally I don't mind including the utility scripts but I worry they may not be wanted in the master branch, and therefore should be kept out of dev as well? @dan-zeman?

I don't know what scripts you are talking about. If they are useful for UD_English-EWT, you can put them somewhere under the `not-to-release` folder, but not in the top-level folder. They will be even propagated to the master branch at release time, but they will not be included in the package we release in Lindat.

If the scripts are useful for multiple treebanks, you can document them and add them to the tools repository.

**Comment by AngledLuffa** [comment]
https://github.com/UniversalDependencies/UD_English-EWT/pull/144/commits/7b95b98a87a94b1d68d4396780d11a670c45f467

The idea being that they either distribute changes from the final conllu files to the input files, or collect the input files into the top level conllu files.  I didn't write them; they were provided by @nschneid , but they seem useful and I'm making another change here anyway.

**Comment by dan-zeman** [comment]
Okay. It seems very useful to have them where you put them, as it is quite difficult to fix anything in UD_English-EWT without them. I'm not aware of other treebanks where the same workflow would be needed, so it is probably not necessary to put them in the tools repo now (they would have to be made more generic anyway; get rid of the ewt-specific paths etc.)

**Comment by AngledLuffa** [comment]
Sounds like we're good to go, then?  I can squash the lemma changes together as needed, or someone else such as @nschneid can commit the scripts themselves if that's desired

**Comment by amir-zeldes** [comment]
FWIW GUM has the same structure in its UD repo, but we don't really need these scripts since both the big and individual document files are generated by a different script from the non-UD GUM source repo.

**Comment by AngledLuffa** [comment]
Also, sneaking in a change to a few gerund lemmas in the dev set

**Comment by AngledLuffa** [comment]
Any further thoughts on this change?  I haven't touched `Islamist` still.  Should I do that?  Should I squash the smaller changes into one big change?

**Comment by nschneid** [comment]
I would capitalize Islamist, yes.

No need to squash yet, I'll do that on merge.

**Comment by AngledLuffa** [comment]
Alright, I made those two changes.

If you're squashing it all together, do you want the scripts in a separate change?

**Comment by amir-zeldes** [comment]
> Should we specify Abbr=Yes for "g-dropped" colloquial forms of -ing verbs?

I'm not really sure it's an abbreviation... isn't it on par with other dialectal and phonological variants? If so, is "gonna" etc. an abbreviation?

**Comment by nschneid** [comment]
> > Should we specify Abbr=Yes for "g-dropped" colloquial forms of -ing verbs?
> 
> I'm not really sure it's an abbreviation... isn't it on par with other dialectal and phonological variants? If so, is "gonna" etc. an abbreviation?

Yes, "gonna", "wanna", and "c'mon" have Abbr=Yes.

I agree that a more specific feature for dialectal/colloquial variant would be nice, but short of that Abbr=Yes seems like a reasonable way to explain why the spelling differs between lemma and wordform.

**Comment by amir-zeldes** [comment]
OK, I can follow that behavior for GUM as well, this should still make it into the upcoming release.

**Comment by nschneid** [comment]
One more thing: I think the 3 gerunds with Abbr=Yes should have a CorrectForm specified in the last column. Even where it's not technically *incorrect* (not a typo), it would be nice to have the normalized inflected form in addition to the lemma.

**Comment by AngledLuffa** [comment]
Updated

File: not-to-release/sources/newsgroup/groups.google.com_civicamerican_22c54c292026bfd2_ENG_20060128_072400.xml.conllu [hunk 1]
```diff
@@ -1077,14 +1077,14 @@
 3	IS	be	AUX	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	cop	4:cop	_
 4	WHY	why	ADV	WRB	PronType=Int	0	root	0:root	_
 5	ALL	all	DET	DT	_	6	det	6:det	_
-6	AMERICANS	AMERICANS	PROPN	NNPS	Number=Plur	8	nsubj	8:nsubj	_
+6	AMERICANS	Americans	PROPN	NNPS	Number=Plur	8	nsubj	8:nsubj	_
```
Comment (nschneid) [inline not-to-release/sources/newsgroup/groups.google.com_civicamerican_22c54c292026bfd2_ENG_20060128_072400.xml.conllu:1080]: lemma = American

File: not-to-release/sources/weblog/blogspot.com_gettingpolitical_20030906235000_ENG_20030906_235000.xml.conllu [hunk 2]
```diff
@@ -31,7 +31,7 @@
 6	ago	ago	ADV	RB	_	4	advmod	4:advmod	_
 7	and	and	CCONJ	CC	_	10	cc	10:cc	_
 8	the	the	DET	DT	Definite=Def|PronType=Art	9	det	9:det	_
-9	israelians	israelians	PROPN	NNPS	Number=Plur	10	nsubj	10:nsubj	_
+9	israelians	Israelians	PROPN	NNPS	Number=Plur	10	nsubj	10:nsubj	_
```
Comment (nschneid) [inline not-to-release/sources/weblog/blogspot.com_gettingpolitical_20030906235000_ENG_20030906_235000.xml.conllu:34]: lemma = Israelian

File: not-to-release/sources/newsgroup/groups.google.com_civicamerican_22c54c292026bfd2_ENG_20060128_072400.xml.conllu (hunk 1 above)
Comment (nschneid) [reply not-to-release/sources/newsgroup/groups.google.com_civicamerican_22c54c292026bfd2_ENG_20060128_072400.xml.conllu:1080]: This is a systematic issue for NNPS. Will move it to another issue.

**Comment by AngledLuffa** [comment]
Do you want me to address it now in this PR?

**Comment by nschneid** [comment]
I'll merge this and then you can do a different PR.

**Pull request merged**